### DISCUSSION TITLE
You are eligible to use GitHub Copilot for free. But vscode require subscription

### DISCUSSION THREAD

**Discussion by ashgaliyev** [community/community#30622] [Copilot Conversations]
I'm confused. In github settings I have this message
<img width="786" alt="Screen Shot 2022-08-24 at 10 20 42 AM" src="https://user-images.githubusercontent.com/818556/186356056-fc1dbbfa-f04e-4c16-be80-0e2eb3851613.png">

But in VSCode I see this and copilot is not working:
<img width="685" alt="Screen Shot 2022-08-24 at 10 21 30 AM" src="https://user-images.githubusercontent.com/818556/186356221-06ba35c9-1ab4-4d08-83e6-5794f7362f50.png">
I don't see where I need to subscribe to copilot. Seems like a bug.

**Comment by ashgaliyev** [comment]
Ok, I just re-installed extension, and it seems working, but still see 'Subscription required' message

**Comment by davecheney** [comment]
Do you have several GitHub accounts?

**Reply by ashgaliyev** [reply]
No. And I just checked to ensure I logged with my current account in VSCode and Github 😁

**Reply by davecheney** [reply]
@ashgaliyev I see your account has access to copilot as an open source maintainer. Please consult the documentation for how to get started with Copilot https://docs.github.com/en/get-started
